plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/jmx/OpenshiftConnectionWrapper.java

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
```

Comment:
Kind of. I copied the impelementation. I will adapt the header to reflect my contribution.